src/viewport.js

```diff
@@ -201,7 +216,7 @@ export default class Viewport {
       pixelUnprojectionMatrix = mat4.invert([], pixelProjectionMatrix);
     }
 
-    const matrices = {
+    const matrices = Object.assign({
```

Comment:
> Why the switch back to Object.assign vs. spread parameter?

I'm making it so that all the repos in our geovisalization stack are pure ES6 - they can run directly in node and the browser without transpilation.

One of the few sacrifices of any importance is support for object spread, unfortunately.